h2o-core/src/test/java/water/fvec/TestFrameBuilderTest.java

```diff
@@ -228,4 +232,33 @@ public void withRandomBinaryDataForColTest(){
 
     fr.delete();
   }
+
+  @Test
+  public void testBadVec(){
+    try {
+      Scope.enter();
+
+      Frame frame = new TestFrameBuilder()
+              .withVecTypes(Vec.T_BAD)
+              .withDataForCol(0, ard(Float.NaN, Float.NaN, Float.NaN)) // All NaN column
+              .withName("fr")
+              .build();
+      Scope.track(frame);
```

Comment:
build() method automatically puts the frame into Scope

Reply:
resolved.